social-tw/social-tw-website
# feat: add close button to adjudicate notification

## Summary

add close button to adjudicate notification

## Linked Issue

close #559 

## Details
- Add a close button on the adjudication floating button.
- Popup the confirm dialog when click the close button.
- Hide the floating button if confirmed, but still show the floating button if there is a new report to adjudicate.

## Impacted Areas

packages/frontend/src/features/reporting/components/AdjudicateNotification

## Verification Steps

close button of notification
1. press close button of the adjudication notification
2. should pop out an dialog of confirmation

confirmation
4. close if pressed confirmed(also closes the notification)
5. jump to adjudication if pressed cancel(also closes the notification)
6. close if pressed close button(does not closes the notification)
